README.md

```diff
@@ -122,6 +122,7 @@ To ease development, you can also use `npm run compile-continuously` to have you
 
 > In this case, you could probably also make use of [`live-server`](https://www.npmjs.com/package/live-server).
```

Comment:
Your `npm run dev` is far superior to `live-server`. Don't you think we should update the doc to remove `compile-continuously` and `live-server`?